contracts/payloads/IGP95/PayloadIGP95.sol

```diff
@@ -0,0 +1,974 @@
+                supplyTokens[0] = USDe_ADDRESS;
+                supplyTokens[1] = USDC_ADDRESS;
+
+                address[] memory borrowTokens = new address[](2);
+                borrowTokens[0] = USDe_ADDRESS;
+                borrowTokens[1] = USDC_ADDRESS;
+
+                // Pause the user operations
+                LIQUIDITY.pauseUser(USDe_USDC_DEX, supplyTokens, borrowTokens);
+            }
+        }
+
+        {
+            address FLUID_ETH_DEX = getDexAddress(10);
+            {
+                // cbBTC-ETH DEX
+                {
+                    setSupplyProtocolLimitsPaused(FLUID_ETH_DEX, FLUID_ADDRESS);
+
+                    setSupplyProtocolLimitsPaused(FLUID_ETH_DEX, ETH_ADDRESS);
+                }
+            }
+
+            {
+                // Pause user supply and borrow
+                address[] memory supplyTokens = new address[](2);
+                supplyTokens[0] = FLUID_ADDRESS;
+                supplyTokens[1] = ETH_ADDRESS;
+
+                address[] memory borrowTokens = new address[](0);
+
+                // Pause the user operations
+                LIQUIDITY.pauseUser(FLUID_ETH_DEX, supplyTokens, borrowTokens);
+            }
+        }
+    }
+
+    // @notice Action 8: Set Launch Limits for RLP-USDC DEX
+    function action8() internal isActionSkippable(8) {
+        address RLP_USDC_DEX = getDexAddress(28);
+        {
+            // RLP-USDC DEX
+            DexConfig memory DEX_RLP_USDC = DexConfig({
+                dex: RLP_USDC_DEX,
+                tokenA: RLP_ADDRESS,
+                tokenB: USDC_ADDRESS,
+                smartCollateral: true,
+                smartDebt: false,
+                baseWithdrawalLimitInUSD: 6_500_000, // $6.5M
+                baseBorrowLimitInUSD: 0, // $0
+                maxBorrowLimitInUSD: 0 // $0
+            });
+            setDexLimits(DEX_RLP_USDC); // Smart Collateral
+
+            DEX_FACTORY.setDexAuth(RLP_USDC_DEX, TEAM_MULTISIG, false);
+        }
+
+        {
+            address fSL28_RLP_USDC = getSmartLendingAddress(28);
+
+            // set rebalancer at fSL28 to reserve contract proxy
+            ISmartLendingAdmin(fSL28_RLP_USDC).setRebalancer(
+                address(FLUID_RESERVE)
+            );
+        }
+        {
+            IFluidDex(RLP_USDC_DEX).updateMaxSupplyShares(
+                5_000_000 * 1e18 // $10M
+            );
+        }
+    }
+
+    // @notice Action 9: Set Launch Limits for Gold Vaults and DEX
+    function action9() internal isActionSkippable(9) {
+        {
+            // PAXG-XAUT DEX
+            address PAXG_XAUT_DEX = getDexAddress(32);
+            {
+                DexConfig memory DEX_PAXG_XAUT = DexConfig({
+                    dex: PAXG_XAUT_DEX,
+                    tokenA: PAXG_ADDRESS,
+                    tokenB: XAUT_ADDRESS,
+                    smartCollateral: true,
+                    smartDebt: false,
+                    baseWithdrawalLimitInUSD: 1_200_000, // $1.2M
```

Comment:
@KaymasJain maybe we want to increase to 5M$ here too for dexes. Or in this case, to 2M$ to have no cap for swap size due to limits